src/models/Review.py

```diff
@@ -0,0 +1,31 @@
+#temp
+from bson import ObjectId
+import datetime
+
+class Review:
+  _id: ObjectId
+  user_id: str
+  drink_id: str
+  comment: set[str] = set() #comments of drinks from this review
+  date: datetime.datetime.now()
+
+  def __init__(self, comment):
```

Comment:
I think it'd be helpful we could pass `user_id` and `drink_id` in the constructor so we don't have to set it manually later on.